agent_code/coli_agent/callbacks.py

```diff
@@ -79,32 +90,48 @@ def state_to_features(game_state, history) -> np.array:
             print(
                 "tried to access tile out of bounds (walls)"
             )  # TODO remove, just for "debugging"
-    features[0] = wall_counter
+    features[0] = wall_counter > 2
```

Comment:
I chose to use an int instead of boolean on purpose because I thought that maybe that would be additional info that can be learned from. But I'm realizing right now that there's never a case where the agent has _one_ adjunct wall, so I guess it doesn't really matter